apps/src/code-studio/assets/SelectStartAnimations.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import {
+  getManifest,
+  generateLevelAnimationsManifest
+} from '@cdo/apps/assetManagement/animationLibraryApi';
+import AnimationPickerBody from '@cdo/apps/p5lab/AnimationPicker/AnimationPickerBody.jsx';
+import {createUuid} from '@cdo/apps/utils';
+import color from '@cdo/apps/util/color';
+import PropTypes from 'prop-types';
+import {PICKER_TYPE} from '@cdo/apps/p5lab/AnimationPicker/AnimationPicker';
+
+const THUMBNAIL_SIZE = 50;
+const THUMBNAIL_BORDER_WIDTH = 1;
+
+export default class SelectStartAnimations extends React.Component {
+  static propTypes = {
+    useAllSprites: PropTypes.bool
+  };
+
+  state = {
+    levelAnimationsManifest: {},
+    libraryManifest: {},
+    orderedKeys: [],
+    propsByKey: {}
+  };
+
+  componentDidMount() {
+    getManifest('spritelab')
+      .then(sprites => {
+        this.setState({libraryManifest: sprites});
+      })
+      .catch(err => {
+        console.log(err);
```

Comment:
nit: we should use `console.error` so we benefit from the styling/formatting that function gives